manifests/m/MariaDB/Server/11.3.2/MariaDB.Server.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: MariaDB.Server
+PackageVersion: 11.3.2
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{60590235-4E76-4AED-91A9-F864EE24D1C4}'

```

```suggestion
ProductCode: '{60590235-4E76-4AED-91A9-F864EE24D1C4}'
ReleaseDate: 2024-02-16
```

Since bot is complaining about it